ProcessMaker/Http/Controllers/Api/Administration/RolesController.php

```diff
@@ -79,4 +79,15 @@ public function create(Request $request)
         return fractal($role, new RoleTransformer())->respond();
     }
 
+    public function update(Request $request, $uid)
+    {
+      $role = Role::where('uid', $uid)->firstOrFail();
+      $role->name=$request->get('name');
+      $role->code=$request->get('code');
+      $role->description=$request->get('description');
+      $role->status=$request->get('status');
+      $role->save();
+      return fractal($role, new RoleTransformer())->respond();
```

Comment:
This needs to refresh the model before sending it or it will only return a partial data set.